src/hooks/useGasPrice.ts

```diff
@@ -77,10 +80,12 @@ const useGasPrice = (): AsyncResult<{
       const maxFee = gasPrice || (isEIP1559 ? feeData?.maxFeePerGas : feeData?.gasPrice) || undefined
       const maxPrioFee = (isEIP1559 && feeData?.maxPriorityFeePerGas) || undefined
 
-      return {
+      const gasEstimation = {
         maxFeePerGas: maxFee,
         maxPriorityFeePerGas: maxPrioFee,
       }
+
+      return adjustGasEstimationForChain(gasEstimation, chain?.chainId)
```

Comment:
Naming-wise, I would call this `gasPriceEstimation` to not confuse it with the gas limit estimation.